<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateDialog</class>
 <widget class="QDialog" name="UpdateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update fields</string>
  </property>
  <property name="windowIcon">
   <iconset theme="edit-find-replace">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="4">
    <widget class="QToolButton" name="allButton">
     <property name="text">
      <string>Show all</string>
     </property>
     <property name="icon">
      <iconset theme="list-add">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="prevButton">
     <property name="text">
      <string>Prev</string>
     </property>
     <property name="icon">
      <iconset theme="previous">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="nextButton">
     <property name="text">
      <string>Next</string>
     </property>
     <property name="icon">
      <iconset theme="next">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="searchEdit">
     <property name="placeholderText">
      <string>Search</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>782</width>
        <height>448</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="fieldsLayout"/>
     </widget>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="clearButton">
     <property name="text">
      <string>Restore</string>
     </property>
     <property name="icon">
      <iconset theme="edit-clear">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>searchEdit</tabstop>
  <tabstop>prevButton</tabstop>
  <tabstop>nextButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>allButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UpdateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UpdateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UpdateDialog</receiver>
   <slot>searchBoxEdited(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prevButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdateDialog</receiver>
   <slot>prevButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdateDialog</receiver>
   <slot>nextButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdateDialog</receiver>
   <slot>clearButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>UpdateDialog</receiver>
   <slot>allButtonChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>594</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>searchBoxEdited(QString)</slot>
  <slot>allButtonChecked(bool)</slot>
  <slot>nextButtonClicked()</slot>
  <slot>prevButtonClicked()</slot>
  <slot>clearButtonClicked()</slot>
 </slots>
</ui>
